src/Exception/NotSupported.php

```diff
@@ -41,4 +43,16 @@ public static function createForPersistence3(string $context): self
             $context,
         ));
     }
+
+    public static function createForPersistence4(string $context): self
```

Comment:
Where is this method called?

Reply:
Nowhere anymore it seems, I'll remove it.